src/components/header/breadCrumbs.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type BreadCrumbsProps = {
+  trail: string; //TODO: Change to real one, propably not a string
```

Comment:
This could already be an array of objects, which it most likely will be in the future. Then the rendering logic could be done also now.

My guess for the data format is:
```javascript
[
  {
    "name": string,
    "url": string,
  },
  ...
]
```